exporter/trace/cloudtrace.go

```diff
@@ -67,41 +67,14 @@ type options struct {
 	// Optional.
 	OnError func(err error)
 
-	// MonitoringClientOptions are additional options to be passed
-	// to the underlying Stackdriver Monitoring API client.
-	// Optional.
-	MonitoringClientOptions []option.ClientOption
```

Comment:
Should this just be renamed to TraceClientOptions instead of removed?  The actual option.ClientOption doesn't appear to be specific to GCM

Reply:
If you look right below this line, there already was a `TraceClientOptions` that had the exact same set of types.